crates/crates_io_trustpub/src/access_token.rs

```diff
@@ -0,0 +1,110 @@
+use secrecy::{ExposeSecret, SecretString};
+use sha2::digest::Output;
+use sha2::{Digest, Sha256};
+
+/// A temporary access token used to publish crates to crates.io using
+/// the "Trusted Publishing" feature.
+#[derive(Debug)]
+pub struct AccessToken(SecretString);
+
+impl AccessToken {
+    const PREFIX: &str = "cio_tp_";
```

Comment:
for regular tokens we use `cio` plus 32 alphanumeric character. when we implemented GitHub Secret Scanning we were told to use a more unique prefix with a separator. therefore the temporary access tokens are using the `cio_to_` prefix to make them easier to recognize with less false positives.